### PR TITLE
Update readme and docstrings

### PR DESCRIPTION
**Pull request by maririos**
- Readme:
  - Improves authentication #9407 and #9461 
  - Adds logging information
  - Removes en-us from links
- Docstrings:
  - Adds example for Key Phrases: #9691 
  - Adds information about data limis.
  - Fixes links to docs

### CODE REVIEW

File: sdk/textanalytics/Azure.AI.TextAnalytics/README.md [hunk 1]
```diff
@@ -13,34 +13,50 @@ Azure Cognitive Services Text Analytics is a cloud service that provides advance
...
 ### Prerequisites
 * An [Azure subscription][azure_sub].
-* An existing [Cognitive Services][cognitive_resource] or Text Analytics resource. If you need to create the resource, you can use the [Azure Portal][azure_portal] or [Azure CLI][azure_cli].
+* An existing Cognitive Services or Text Analytics resource.
 
-If you use the Azure CLI, replace `<your-resource-group-name>`, `<your-resource-name>`, `<location>`, and `<sku>` with your values:
+### Create a Cognitive Services or Text Analytics resource
+Text Analytics supports both [multi-service and single-service access][cognitive_resource_portal]. Create a Cognitive Services resource if you plan to access multiple cognitive services under a single endpoint/key. For Text Analytics access only, create a Text Analytics resource.
+
+You can create either resource using the [Azure Portal][cognitive_resource_portal] or [Azure CLI][cognitive_resource_cli]. Below is an example of how you can create a Text Analytics resource using the CLI:
```
Comment (kristapratico) [inline sdk/textanalytics/Azure.AI.TextAnalytics/README.md]: Let's do the Option 1 or Option 2 thing we talked about in the meeting today (see my PR)

File: sdk/textanalytics/Azure.AI.TextAnalytics/README.md [hunk 2]
```diff
@@ -13,34 +13,50 @@ Azure Cognitive Services Text Analytics is a cloud service that provides advance
...
 ```PowerShell
-Install-Package Azure.AI.TextAnalytics
+Install-Package Azure.AI.TextAnalytics -Version 1.0.0-preview.2
 ```
 
 ### Authenticate the client
 In order to interact with the Text Analytics service, you'll need to create an instance of the [TextAnalyticsClient][textanalytics_client_class] class. You will need an **endpoint**, and either an **API key** or ``TokenCredential`` to instantiate a client object.  For more information regarding authenticating with cognitive services, see [Authenticate requests to Azure Cognitive Services][cognitive_auth].
 
 #### Get API Key
+You can get the `endpoin`t and `API key` from the Cognitive Servicesresource or Text Analytics resource information in the [Azure Portal][azure_portal].
```
Comment (kristapratico) [inline sdk/textanalytics/Azure.AI.TextAnalytics/README.md]: The `t` in endpoint got cut off here

**Review by kristapratico — APPROVED** [review]

File: sdk/textanalytics/Azure.AI.TextAnalytics/README.md (hunk 1 above)
Comment (mssfang) [inline sdk/textanalytics/Azure.AI.TextAnalytics/README.md]:
```suggestion
You can create either resource using the **Option 1:** [Azure Portal][cognitive_resource_portal] or **Option 2:** [Azure CLI][cognitive_resource_cli]. Below is an example of how you can create a Text Analytics resource using the CLI:
```

File: sdk/textanalytics/Azure.AI.TextAnalytics/README.md [hunk 3]
```diff
@@ -13,34 +13,50 @@ Azure Cognitive Services Text Analytics is a cloud service that provides advance
...
+# Create text analytics
+az cognitiveservices account create \
+    --name <your-resource-name> \
+    --resource-group <your-resource-group-name> \
+    --kind TextAnalytics \
+    --sku <sku> \
+    --location <location> \
+    --yes
 ```
+For more information about creating the resource or how to get the location and sku information see [here][cognitive_resource_cli].
```
Comment (mssfang) [inline sdk/textanalytics/Azure.AI.TextAnalytics/README.md]: Does it duplicate what [Azure CLI][cognitive_resource_cli] does at above, https://github.com/Azure/azure-sdk-for-net/pull/9887/files#diff-c183227058ebd27d819132b62a437f6eR21

File: sdk/textanalytics/Azure.AI.TextAnalytics/README.md [hunk 4]
```diff
@@ -81,12 +97,13 @@ A `TextAnalyticsClient` is the primary interface for developers using the Text A
...
 
 ### Operation Result Collection
-An operation result collection, such as `AnalyzeSentimentResultCollection`, is a collection of operation results, where each corresponds to one of the text inputs provided in the input batch.  A text input and its result will have the same index in the input and result collections.  An operation result collection may optionally include information about the input batch and how it was processed.
+An operation result collection, such as `AnalyzeSentimentResultCollection`, is a collection of operation results, where each corresponds to one of the text inputs provided in the input batch.  A text input and its result will have the same index in the input and result collections.  An operation result collection contains a `HasError` property that allows to identify if an operetation executed was succesful or unsuccesful for the given text input. It may optionally include information about the input batch and how it was processed.
```
Comment (mssfang) [inline sdk/textanalytics/Azure.AI.TextAnalytics/README.md]: I think it is "TextAnalyticsResult" contains a  HasError property?

Comment (mssfang) [reply sdk/textanalytics/Azure.AI.TextAnalytics/README.md]: typo: operetation

File: sdk/textanalytics/Azure.AI.TextAnalytics/README.md (hunk 2 above)
Comment (kaerm) [inline sdk/textanalytics/Azure.AI.TextAnalytics/README.md]: and the Servicesresource is a one word

File: sdk/textanalytics/Azure.AI.TextAnalytics/README.md [hunk 5]
```diff
@@ -334,26 +362,29 @@ This project has adopted the [Microsoft Open Source Code of Conduct][code_of_con
...
 [textanalytics_nuget_package]: https://www.nuget.org/packages/Azure.AI.TextAnalytics
 [textanalytics_samples]: https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/textanalytics/Azure.AI.TextAnalytics/tests/samples
 [textanalytics_rest_api]: https://westus.dev.cognitive.microsoft.com/docs/services/TextAnalytics-v3-0-Preview-1/operations/Languages
-[cognitive_resource]: https://docs.microsoft.com/en-us/azure/cognitive-services/cognitive-services-apis-create-account
-
-[language_detection]: https://docs.microsoft.com/en-us/azure/cognitive-services/Text-Analytics/how-tos/text-analytics-how-to-language-detection
-[sentiment_analysis]: https://docs.microsoft.com/en-us/azure/cognitive-services/Text-Analytics/how-tos/text-analytics-how-to-sentiment-analysis
-[key_phrase_extraction]: https://docs.microsoft.com/en-us/azure/cognitive-services/Text-Analytics/how-tos/text-analytics-how-to-keyword-extraction
-[named_entity_recognition]: https://docs.microsoft.com/en-us/azure/cognitive-services/Text-Analytics/how-tos/text-analytics-how-to-entity-linking
+[cognitive_resource_portal]: https://docs.microsoft.com/azure/cognitive-services/cognitive-services-apis-create-account
```
Comment (kaerm) [inline sdk/textanalytics/Azure.AI.TextAnalytics/README.md]: Should we use the links that Mary talked about to go directly to resource creation in portal? https://ms.portal.azure.com/#create/Microsoft.CognitiveServicesTextAnalytics

**Review by kaerm — APPROVED** [review]

File: sdk/textanalytics/Azure.AI.TextAnalytics/README.md (hunk 3 above)
Comment (maririos) [reply sdk/textanalytics/Azure.AI.TextAnalytics/README.md]: It is the same link, but I wanted to make it obvious that there is information on it about how to get location and sku

File: sdk/textanalytics/Azure.AI.TextAnalytics/README.md (hunk 5 above)
Comment (maririos) [reply sdk/textanalytics/Azure.AI.TextAnalytics/README.md]: I prefer guiding the user to the service documentation on how to do it and from there, they can decide if that is for them.
Sending them to that link implies that the user has already an account with Azure, authenticated, etc.

**Pull request merged**